elasticsearch/sandboxes/heka/output/elasticsearch_bulk_api.lua

```diff
@@ -142,8 +148,46 @@ local function send_batch()
     return ret, err
 end
 
-batch_count = 0
-retry       = false
+local batch_count = 0
+local retry       = false
+local try_count   = 0
+
+for _ in io.lines(batch_file) do
+    batch_count = batch_count + 1
+end
+batch_count = batch_count / 2
```

Comment:
Why do we need this division? Elasticsearch batch data takes 2 lines per message?